ShadowsocksX-NG/example-gui-config.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
this file is add to the project here, should check if the file is not copy to resource bundle in build